awx/main/tests/functional/analytics/test_counts.py

```diff
@@ -12,7 +12,6 @@ def test_empty():
         "active_sessions": 0,
         "active_host_count": 0,
         "credential": 0,
-        "custom_inventory_script": 0,
```

Comment:
@rooftopcellist I'm not yet sure if what I've done here works with analytics. It would be trivial to keep these keys but return a count of 0 every time (so that the schema doesn't change). I just need to figure out how to test and assure that it still works.